adal/src/androidTest/java/com/microsoft/aad/adal/DefaultTokenCacheStoreTests.java

```diff
@@ -122,12 +122,12 @@ protected StorageHelper getStorageHelper() {
         return cache;
     }
 
-    
+
     public void testDateTimeFormatterLocaleChange() throws AuthenticationException {
-        DefaultTokenCacheStore store = (DefaultTokenCacheStore)setupItems();
+        DefaultTokenCacheStore store = (DefaultTokenCacheStore) setupItems();
         List<TokenCacheItem> tokens = store.getTokensForResource("resource");
         // Serializing without miliseconds
-        long precision = 1000;
+        final long precision = 1000;
```

Comment:
since you did this. All the DefaultTokenCacheStore objects in the rest of this file can be final too.